addons/hr/static/src/js/hr_employee.js

```diff
@@ -108,8 +108,30 @@ odoo.define('hr.employee_chat', function (require) {
                 }
             }
             return actionMenuItems;
-        }
+        },
+
+        /**
+         * When we run Launch Plan list action, in case if we close the
+         * wizzard, the originally selected employees stay selected.
+         * 
+         * @override
+        */
+        async _executeButtonAction(actionData, recordData) {
+            // I guess this needs translation?
+            let is_launch_action = actionData.string == _t("Launch Plan");
```

Comment:
Relying on the `.string` seems a bit fragile, maybe you can add a `class` on the button and use that instead (I'm not sure what you are getting in actionData tho)? :thinking: